pandas/core/api.py

```diff
@@ -10,6 +10,7 @@
 )
 from pandas.core.dtypes.missing import isna, isnull, notna, notnull
 
+# TODO: Remove get_dummies import when statsmodels updates #18264
 from pandas.core.algorithms import factorize, unique, value_counts
 from pandas.core.arrays import Categorical
 from pandas.core.arrays.integer import (
```

Comment:
future, we should have a pandas.core.arrays.api that we can just import here

Reply:
We have `pandas.core.arrays.__init__` which serves this purpose.

Reply:
IIRC the presence of imports in `arrays.__init__` has caused some fragility with circular imports that could be avoided by putting these in arrays.api